packages/cli/src/ui/commands/clearCommand.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SlashCommand } from './types.js';
+
+export const clearCommand: SlashCommand = {
+  name: 'clear',
+  description: 'clear the screen and conversation history',
+  action: async (context, _args) => {
+    context.utils.onDebugMessage('Clearing terminal and resetting chat.');
+    context.ui.clearItems();
```

Comment:
```suggestion
    context.ui.clear();
```

And we should probably make ui.clear() do the `console.clear()` and `refreshStatic` too. Honestly I might recommend dropping `refreshStatic` entirely from the ui context piece because it's a bit of a detail